src/scss/objects/_form-toggle.scss

```diff
@@ -0,0 +1,166 @@
+/**
+ * Form Toggles
+ * --------------------------------
+ * Used for listing page filters
+ *
+ * ================================= */
+
+$formToggle-padding                 : spacing() !default;
+$formToggle-border-color            : $grey--lightest;
+$formToggle-border-color-checked    : $grey--lighter;
+$formToggle-border-width            : 1px;
+$formToggle-border-radius           : 8px;
+$formToggle-tick-background         : $green;
+$formToggle-background              : $white;
+$formToggle-large-size              : 48px;
+$formToggle-button-color            : $blue;
+$formToggle-button-background       : $blue--offWhite;
+
+.o-formToggle {
+    position: relative;
+    display: inline-block;
+    margin-bottom: spacing();
+    border-radius: $formToggle-border-radius;
+    padding: ($formToggle-padding / 2) $formToggle-padding;
+    border: $formToggle-border-width solid $formToggle-border-color;
+
+    @include media('>=mid') {
+        background: $formToggle-background;
+    }
+}
+
+    .o-formToggle--button {
+        padding-left: spacing(x4);
+        color: $formToggle-button-color;
+        background: $formToggle-button-background;
+        border-color: $formToggle-button-background;
+    }
+
+    .o-formToggle--buttonIcon {
+        top: 50%;
+        position: absolute;
+        left: $formToggle-padding;
+        transform: translateY(-50%);
+        fill: $formToggle-button-color;
+    }
+
+    .o-formToggle--fullWidth {
+        width: 100%;
+        display: block;
+
+        @include media('>=mid') {
+            display: inline-block;
+        }
+    }
+
+    .o-formToggle--withCount {
+        padding-right: 15px;
+    }
+
+    .o-formToggle--withBg {
+        background: $formToggle-background;
+    }
+
+    .o-formToggle--mediumSized {
+        padding: $formToggle-padding ($formToggle-padding * 1.5);
+    }
+
+    .o-formToggle--largeSized {
+        height: $formToggle-large-size;
+        width: calc(50% - 4px);
+
+        &:nth-child(2n) {
+            margin-left: spacing();
+        }
+    }
+
+    .o-formToggle--largeAboveMid {
+
+        @include media('>=mid') {
+            float: left;
+            width: calc(50% - 4px);
```

Comment:
Would be good to simplify this and the styles in `.o-formToggle--largeSized` so that they don't have duplicate styles, but share the same property defs.